packages/hint/src/lib/utils/schema-validator.ts

```diff
@@ -6,7 +6,7 @@ import {
     without
 } from 'lodash';
 
-import { SchemaValidationResult } from '../types';
+import { ProblemLocation, IJSONLocationFunction, ISchemaValidationError, SchemaValidationResult } from '../types';
```

Comment:
`ProblemLocation` is not used.

Fixed in https://github.com/webhintio/hint/commit/a1e1e51c658e7bcbba7bb4d5db56c95c5731e494.